server/src/test/java/org/elasticsearch/index/mapper/TextFieldMapperTests.java

```diff
@@ -1190,6 +1190,18 @@ protected Function<Object, Object> loadBlockExpected() {
         return v -> ((BytesRef) v).utf8ToString();
     }
 
+    protected boolean nullLoaderExpected(MapperService mapper, String fieldName) {
```

Comment:
Probably want `@Override` for this one.

Reply:
Maybe we should move this logic into `loadBlockExpected` and add the required parameters.

Reply:
It makes sense

My only doubt is: should I consider `MapperTestCase.loadBlockExpected()` a public API? I know it's in tests, but these are standard tests for field mappers, and this change will likely break the codebase of third party plugins implementing custom data types. 

WDYT?

Reply:
Nah, it's fine to change `MapperTestCase` - we do it all the time. I think I did it last week.

Plugins indeed might use it for mapped field types, but if they want to upgrade they'll have to handle all kinds of changes there. We're really not stable here. In some sense that's good because their upgrades will make them think about things like synthetic _source. But in other senses it's bad because it'll break compilation. But, at least for now, we break.

Reply:
i'd switch this method to `private` now that it's only used locally.